build.gradle

```diff
@@ -148,33 +149,60 @@ tasks.register('checkAkitInstall', JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
 }
 
-tasks.register('installLocalGitHookPostCheckout', Copy) {
-    from new File(rootProject.rootDir, '.scripts/post-checkout')
-    into { new File(rootProject.rootDir, '.git/hooks') }
-    filePermissions {
-        unix(0775)
-    }
-    doLast {
-        println("---Added Post Checkout Git Hook Script---")
+def ensureSymlink(link, target) {
```

Comment:
```suggestion
def ensureSymbolicLink(link, target) {
```

Reply:
I don't really care for this; you can commut it if you want

Reply:
commit suggestion doesn't refactor